search/populate.py

```diff
@@ -0,0 +1,52 @@
+from io import BytesIO
+
+from flask import Blueprint, Response
+from flask import current_app as app
+
+import requests
+from elasticsearch import Elasticsearch
+from lxml import etree
+
+from search.utils import strip_slashes
+
+
+def get_populate_blueprint() -> Blueprint:
+    blueprint = Blueprint('populate', __name__)
+
+    @blueprint.route('/populate', methods=['POST'])
+    def populate() -> Response:
+        """
+        Route for triggering indexing of known paths. Will be used as a proof of
+        concept. This should be removed before MVP release.
+        """
+        search = Elasticsearch(hosts=app.config['ELASTICSEARCH_HOSTS'])
+        gateway = strip_slashes(app.config['GATEWAY_URL'])
+
+        for path in app.config['PATHS_TO_INDEX']:
+            url = f'{gateway}/{strip_slashes(path)}'
+            response = requests.get(url)
```

Comment:
Will need to cover pagination (https://docs.google.com/document/d/1JhVpW8dxAuJqnk2YsQa76yxdIV_BlOd1QIi02fIu9mo/edit#heading=h.jc8ntuvi60cs).

Reply:
Issue [95](https://github.com/libero/libero/issues/95) doesn't state to handle pagination.

Reply:
It's in the spec.